### PR TITLE
separate out fs plugin, mod plugin order

### PR DESCRIPTION
**Pull request by jenius**
:cactus:

### CODE REVIEW

File: lib/plugins/fs_plugin.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import util from './plugin_utils'
+
+export default class JadeWebpackPlugin {
+
+  // Accepted Options
+  // - matcher (string)
+  // - dumpDirs (array)
```
Comment (kylemac) [inline lib/plugins/fs_plugin.js]: update this to be `matchers` and `ignore`

File: lib/plugins/fs_plugin.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  constructor (opts) {
+    this.opts = opts
+  }
+
+  apply (compiler) {
+    // read file tree and get all jade files
```
Comment (kylemac) [inline lib/plugins/fs_plugin.js]: > read file tree and get all ~~jade~~ files

**Comment by kylemac** [comment]
@jenius two minor comment comments and then :+1:

**Comment by kylemac** [comment]
made the updates and merged

**Comment by jenius** [comment]
thank you! :+1:

**Pull request merged**